assets/javascripts/discourse/initializers/apply-saved-searches.js.es6

```diff
@@ -1,14 +1,15 @@
 import { withPluginApi } from "discourse/lib/plugin-api";
 import discourseComputed from "discourse-common/utils/decorators";
+import { helperContext } from "discourse-common/lib/helpers";
 
 function initializeSavedSearches(api) {
+  let siteSettings = helperContext().siteSettings;
```

Comment:
In an an initializer we prefer to look it up on the container. 
`let siteSettings = container.lookup("site-settings:main");`